File: python/ejer_51.py
# 51. Registrar 5 notas, calcular promedio y determinar estado
notes = [8.5, 9.0, 7.5, 8.0, 8.2]
avg = sum(notes) / len(notes)
status = 'Aprobado' if avg >= 7 else 'Reprobado'
print('Notas:', notes)
print('Promedio:', round(avg, 2))
print('Estado:', status)

File: js/ejer_51.js
// 51. Registrar 5 notas, calcular promedio y determinar estado
const notes = [8.5, 9.0, 7.5, 8.0, 8.2];
const avg = notes.reduce((a, n) => a + n, 0) / notes.length;
const status = avg >= 7 ? 'Aprobado' : 'Reprobado';
console.log('Notas:', notes);
console.log('Promedio:', avg.toFixed(2));
console.log('Estado:', status);
